src/components/Training/course/CourseCloseButton.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import close from '../../../images/close.svg'
+
+const Close = styled(Link)`
+  border: none;
+  border-radius: 50%;
+  width: ${remcalc(54)};
+  height: ${remcalc(54)};
+  background-color: ${props => props.theme.colors.text};
+  right: ${remcalc(24)};
+  top: ${remcalc(36)};
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+
+  &:focus {
+    outline: ${remcalc(4)} solid ${props => props.theme.colors.vibrant};
+  }
+
+  &:active {
+    outline: none;
+    background-color: ${props => props.theme.colors.vibrant};
+
+    &:after {
+      background: ${props => props.theme.colors.text};
+    }
+  }
+
+  ${breakpoint('smallTablet')`
+    right: ${remcalc(42)};
+  `}
+
+  ${breakpoint('tablet')`
+    right: ${remcalc(48)};
+  `}
+
+   ${breakpoint('desktop')`
+    right: ${remcalc(90)};
+  `}
+`
+
+const CourseCloseButton = ({ closeTo }) => (
+  <Close to={closeTo}>
```

Comment:
not to be nitpicking but I'm not a big fan of `closeTo` as a prop name. If we're already using a `CloseButton` component, having a "close" preffix on the prop names is a bit unnecessary, don't you think?

Reply:
sure :+1: